test/plugins/test/dragresize.js

```diff
@@ -0,0 +1,116 @@
+(function() {
+    'use strict';
+
+    var assert = chai.assert;
+
+    describe('imageScaleResize', function() {
+
+        var url = 'http://localhost/url_test';
+
+        describe('default imageScaleResize "both"', function() {
+            this.timeout(35000);
+
+            before(function(done) {
+                Utils.createCKEditor.call(this, done, {extraPlugins: 'ae_dragresize'});
+            });
+
+            after(Utils.destroyCKEditor);
+
+            beforeEach(Utils.beforeEach);
+
+            afterEach(Utils.afterEach);
+
+            it('The image, that is selected, has all resize handlers', function() {
+                bender.tools.selection.setWithHtml(this.nativeEditor, 'Test image dragresize plugin {<img src="url_test" />} here.');
+
+                var handlers = document.getElementsByTagName('i');
+
+                assert.strictEqual(handlers.length, 8);
+            });
+
+            it('Preview is created, where backgroundImage is equal to src of the image, when mousedown is fired by some handle', function(done) {
+                bender.tools.selection.setWithHtml(this.nativeEditor, 'Test image dragresize plugin {<img src="' + url + '" />} here.');
+
+                var handlers = document.getElementsByTagName('i');
+
+                happen.mousedown(handlers[0]);
+
+                setTimeout(function() {
+                    var span = document.getElementsByTagName('span')[0];
+
+                    assert.strictEqual('url("' + url + '")', span.style.backgroundImage);
+
+                    done();
+                }.bind(this), 0);
+            });
+        });
+
+        describe('"scale" as imageScaleResize value', function() {
+            this.timeout(35000);
+
+            before(function(done) {
+                Utils.createCKEditor.call(this, done, {extraPlugins: 'ae_dragresize', imageScaleResize: 'scale'});
+            });
+
+            after(Utils.destroyCKEditor);
+
+            beforeEach(Utils.beforeEach);
+
+            afterEach(Utils.afterEach);
+
+            it('The image, that is selected, has 4 handlers', function() {
```

Comment:
The sentence must start with the word 'should', for example:
'should add 4 handlers to a selected image'